Sources/SnowManLogicSpecs/GameSpec.swift

```diff
@@ -0,0 +1,67 @@
+import SnowManLogic
+import Swiftest
+
+class GameSpec: Swiftest.Spec {
+	let spec = describe("game logic") {
+		var guessManager: GuessManager!
+		var word: String!
+		var io: MockIO!
+		var view: View!
+		var game: Game!
+
+		before() {
+			guessManager =  GuessManager(totalIncorrectGuessesAllowed: 5)
+			word = "apple"
+			io = MockIO()
+			view = View(io: io)
```

Comment:
Rather than wiring up a view with the MockIO, you should be using the MockView to make sure that the view's interface is being used as expected